signer/apk2/apk2.go

```diff
@@ -119,11 +119,12 @@ func (s *APK2Signer) SignFile(file []byte, options interface{}) (signer.SignedFi
 	defer os.Remove(tmpAPKFile.Name())
 	ioutil.WriteFile(tmpAPKFile.Name(), file, 0755)
 
-	apkSigCmd := exec.Command("java", "-jar", "/usr/bin/apksigner", "sign",
```

Comment:
`/usr/bin/apksigner` is a shell script. No idea why we were calling it as a jar file before.